ios/SnapchatShare.m

```diff
@@ -0,0 +1,180 @@
+//
+//  SnapchatShare.m
+//  RNShare
+//
+//  Created by VincentRoest
+//
+
+#import "SnapchatShare.h"
+#import <AVFoundation/AVFoundation.h>
+@import SCSDKCreativeKit;
+@import Photos;
+
+@implementation SnapchatShare {
+    SCSDKSnapAPI *_scSdkSnapApi;
+}
+
+RCT_EXPORT_MODULE();
+
+-(id)init
+{
+    self = [super init];
+    if(self)
+    {
+        if (!_scSdkSnapApi) {
+            _scSdkSnapApi = [[SCSDKSnapAPI alloc] init];
+            NSLog(@"hi");
+        }
+    }
+    return self;
+}
+
+- (void *)shareSingle:(NSDictionary *)options
+    failureCallback:(RCTResponseErrorBlock)failureCallback
+    successCallback:(RCTResponseSenderBlock)successCallback {
+
+    NSLog(@"Try open view");
```

Comment:
I'm not sure if other share options directly log but I don't think that's good practice? At minimum if you do log it should contain enough information to know exactly where to locate the log for problem localization, e.g. 'RNShare::shareSingle - Try open view'